scripts/ci-conformance.sh

```diff
@@ -69,7 +83,8 @@ if grep -iqF "$(echo "${AWS_ACCESS_KEY_ID-}" | \
   exit 1
 fi
 
-hack/ci/e2e-conformance.sh --verbose
+# TODO(dims): remove "SKIP_INIT_IMAGE" once we fix problem building images in CI
```

Comment:
The images are built using kubernetes master?

Reply:
by default we pick up images with k8s 1.16.2